dict/P/pandunia.tsv

```diff
@@ -719,6 +725,7 @@ PWN:field.n.01		medan	medan	ara: مَيْدَان‎ (maydān), fas:(meydan), hi
 PWN:field_hockey.n.01		hoki	hoki	
 PWN:fight.v.03		jihad	jihad	
 PWN:filming.n.01		kinografia	kin·o·graf·ia	
+PWN:find.v.03		eureka	eureka	eng:eureka, spa:eureka, rus:э́врика (évrika), tur:evreka, cmn:尤里卡, jpn:ユーレカ

```

Comment:
This is a good old Pandunia word. :) However, there is now new form _heure_ at `PWN:invent.v.01`.